src/coll_score/ucc_coll_score.c

```diff
@@ -131,7 +132,8 @@ static ucc_status_t ucc_fallback_alloc(ucc_score_t              score,
 }
 
 static inline void ucc_fallback_insert(ucc_list_link_t  *list,
-                                       ucc_coll_entry_t *fb)
+                                       ucc_coll_entry_t *fb,
+                                       int free_fb)
```

Comment:
seems always called with free_fb = 1. is it needed?

Reply:
not really needed, maybe only to explicitly say that this function will remove object
@vspetrov wdyt?

Reply:
@Sergei-Lebedev sounds good to me

Reply:
@bureddy fixed